src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -70,10 +72,15 @@ const tracesSampleRate = config ? config.apmSampling : 0;
 const hasReplays =
   window.__SENTRY__USER && window.__SENTRY__USER.isStaff && !!process.env.DISABLE_RR_WEB;
 
+const appRoutes = Router.createRoutes(routes());
```

Comment:
For some reason, I cannot move this to `src/sentry/static/sentry/app/utils/apm.tsx`

Reply:
What happens if you move it?

Reply:
The app doesn't load.